.env.example

```
@@ -0,0 +1,7 @@
+DATABASE_DEV_URL=''
+DATABASE_TEST_URL=''
+JWT_SECRET=''
+FACEBOOK_APP_ID=''
+FACEBOOK_APP_SECRET=''
+GOOGLE_APP_ID=''
+GOOGLE_APP_SECRET=''
```

Add a blank line at the end of the file
```suggestion
GOOGLE_APP_SECRET=''

```